.circleci/config.yml

```diff
@@ -664,3 +1267,28 @@ workflows:
     jobs:
       - build:
           context: nightly_env
+
+  nightly-workflows:
+    triggers:
+      - schedule:
+          # should trigger every day at 20:00 PM UTC (11 PM Israel Time)
+          cron: "0 20 * * *"
+          filters:
+            branches:
+              only:
+                - config_yaml
```

Comment:
shouldn't this be the name of the current branch? or if you plan to merge it - `master`

Reply:
There are a lot more changes in config_yaml branch that cannot be merged to master and still maintain the current behavior, so I thought it's best to merge only the config_yaml changes and redirect the actual build to my branch